modules/apps/commerce/commerce-product-asset-categories-web/src/main/resources/META-INF/resources/js/EditAssetCategoryCPDisplayLayout.js

```diff
@@ -21,41 +21,43 @@ export default function ({
 	locale,
 	portletNamespace,
 }) {
-	const displayPageItemRemove = window.document.querySelector(
-		`#${portletNamespace}displayPageItemRemove`
+	const displayPageItemRemove = document.getElementById(
+		`${portletNamespace}displayPageItemRemove`
 	);
 
-	const displayPageNameInput = window.document.querySelector(
-		`#${portletNamespace}displayPageNameInput`
+	const displayPageNameInput = document.getElementById(
+		`${portletNamespace}displayPageNameInput`
 	);
 
-	const pagesContainerInput = window.document.querySelector(
-		`#${portletNamespace}pagesContainerInput`
+	const pagesContainerInput = document.getElementById(
+		`${portletNamespace}pagesContainerInput`
 	);
 
-	window.document
-		.querySelector(`#${portletNamespace}chooseDisplayPage`)
-		.addEventListener('click', (_event) => {
-			openSelectionModal({
-				buttonAddLabel: Liferay.Language.get('done'),
-				onSelect: (selectedItem) => {
-					if (
-						selectedItem &&
-						displayPageNameInput &&
-						pagesContainerInput
-					) {
-						pagesContainerInput.value = selectedItem.id;
-
-						displayPageNameInput.innerHTML = selectedItem.name;
-
-						displayPageItemRemove?.classList.remove('hide');
-					}
-				},
-				selectEventName: 'selectDisplayPage',
-				title: Liferay.Language.get('select-category-display-page'),
-				url: itemSelectorUrl,
-			});
+	const chooseDisplayPageButton = document.getElementById(
+		`${portletNamespace}chooseDisplayPage`
+	);
+
+	chooseDisplayPageButton?.addEventListener('click', () => {
+		openSelectionModal({
+			buttonAddLabel: Liferay.Language.get('done'),
```

Comment:
Is this single or multiple selection?
- if multiple, we are missing `multiple: true`
- if single, we can remove `buttonAddLabel`. This would probably be a change of UI behavior, so we need to warn the product team.

Reply:
It's single selection! You're right about removing it. I'll have to double check how it was before my changes, to see if they even displayed the footer. 👍